proposals/infra/INF-0005-register-types-and-diagnostics.md

```diff
@@ -130,21 +130,275 @@ is no corresponding member, a warning will be emitted that is treated as an erro
 because this behavior was permissible in legacy versions of the compiler. These warnings
 are also part of the same warning group, `LegacyConstantRegisterBinding`.
 
-`DiagnoseHLSLRegisterAttribute` will also be responsible for emitting a diagnostic
-if any other invalid register type is detected. If `DiagnoseHLSLRegisterAttribute`
-finds any critical errors, the attribute, `HLSLResourceBindingAttr`, won't be added
+`DiagnoseHLSLRegisterAttribute` will also be responsible for emitting a diagnostic 

```

Comment:
There's something strange about the line endings (maybe) in this file?  These diffs are odd.  Plus if I try and use the "Add a suggestion" feature then the diff thinks that the entire line has changed.

Any ideas what may have happened?

Reply:
I think I just left some whitespace originally, merged that in, and then Chris opened a PR to fix it while this new PR was up.